docker-compose.yml

```diff
@@ -163,6 +163,24 @@ services:
             - ./docker/haproxy/cors.lua:/etc/haproxy/cors.lua
         scale: -1
 
+    loki:
+      image: grafana/loki:2.4.0
+      volumes:
+        - ./docker/loki:/etc/loki
+        - ${CEPH_REPO_DIR}/build.ceph/out:/etc/logs
+      ports:
+        - "3100:3100"
+      command: '--config.file=/etc/loki/loki-config.yaml'
+
+    promtail:
+      image: grafana/promtail:2.4.0
+      volumes:
+        - /var/log:/var/log
```

Comment:
Do we really need to share the host whole log folder or can we share a more specific folder?